frontend/src/features/Train/features/ObjectDetection/constants/detectionConstants.ts

```diff
@@ -0,0 +1,232 @@
+import DetectionImageStep from "../components/DetectionImageStep";
+import DetectionParametersStep from "../components/DetectionParametersStep";
+import DetectionReviewStep from "../components/DetectionReviewStep";
+import TabularTrainspace from "../components/DetectionTrainspace";
+
+export const TRAINSPACE_SETTINGS = {
+  name: "Detection",
+  steps: ["IMAGE", "PARAMETERS", "REVIEW"],
+  component: TabularTrainspace,
+} as const;
+
+export const STEP_SETTINGS = {
+  IMAGE: {
+    name: "Image",
+    optional: false,
+    component: DetectionImageStep,
+  },
+  PARAMETERS: {
+    name: "Parameters",
+    optional: false,
+    component: DetectionParametersStep,
+    detectionTypes: [
+      { label: "Rekognition", value: "rekognition" },
+      { label: "YOLO", value: "yolo" },
+    ],
+    detectionProblemTypes : [
+      { label: "Labels", value: "labels" },
+      { label: "Celebrities", value: "celebrities" },
+    ],
+    detectionTransformValues: ["RandomHorizontalFlip", "ToTensor", "RandomVerticalFlip", "Resize", "GaussianBlur", "Grayscale", "Normalize", "AdjustContrast", "AdjustBrightness", "Affine"],
+    detectionTransforms: {
+      RandomHorizontalFlip: {
+        display_name: "Random Horizontal Flip",
+        objectName: "transforms.RandomHorizontalFlip",
+        parameters: [
+          {
+            label: "probability",
+            index: 0,
+            parameter_name: "prob",
```

Comment:
What is parameter_name for btw? It doesn't seem to be referenced in your object detection code

Reply:
I think its just a parameter from old components that I wasn't sure whether to remove or not. It does the same thing as label so I've removed it.